src/adl.cpp

```diff
@@ -2435,7 +2436,8 @@ bool CadlPlayer::load(const std::string &filename, const CFileProvider &fp)
   file_data = new uint8 [file_size];
   f->readString((char *)file_data, file_size);
 
-  _driver->callback(8, int(-1));
+  // Moved into rewind.
+  //_driver->callback(8, int(-1));
```

Comment:
You can just remove these comments, they're obvious and can be easily seen in the patch diff.

Reply:
Done.